### PR TITLE
Update Tech Stack on open source page

### PR DESCRIPTION
**Pull request by KnightRadiant44**
### Description
Update few tech stacks on open source page

### Changes Made
Added the necessary tech stack requested.

### Related Issues
Fixes #169 

### Additional Notes
image

### CODE REVIEW

**Comment by KnightRadiant44** [comment]
It worked!! Apologies for the mistakes earlier, I didn't mean to do that. This was a really good learning experience for me :))

File: dump.rdb
Comment (rayokamoto) [inline dump.rdb:1]: This file shouldn't be here. Remember not to `git add .` without first checking what files were changed (e.g. with `git status`). Check what files are being added before you stage your changes and commit.

File: src/constants/colours.ts [hunk 1]
```diff
@@ -31,6 +31,10 @@ export const TECH_COLORS = {
     Supabase: '#5DBBB5',
     'Tailwind CSS': '#907FC3',
     TypeScript: '#ACCB00',
+    Requests: '#FFD166',
+    Vite: '#646464',
+    'Pragmatic Drag & Drop': '#FF6F61',
+    NextUI: '#A8DADC',
```
Comment (rayokamoto) [inline src/constants/colours.ts:37]: Could you order this alphabetically within the `TECH_COLORS` list?

File: src/constants/tech-stack.ts [hunk 2]
```diff
@@ -7,5 +7,9 @@ export const TECH_STACK = [
     'Tailwind CSS',
     'Supabase',
     'Discord.py',
+    'Requests',
+    'Vite',
+    'Pragmatic Drag & Drop',
+    'NextUI',
```
Comment (rayokamoto) [inline src/constants/tech-stack.ts:13]: Could you order this alphabetically? I know the current list isn't either but it'll be nicer if the entire list was ordered.

**Review by rayokamoto — CHANGES_REQUESTED** [review]

File: src/constants/colours.ts [hunk 3]
```diff
@@ -22,15 +22,20 @@ export const BORDER_COLOURS = {
...
+// the list below is sorted in alphabetical order by tech stack name
 export const TECH_COLORS = {
     'Discord.py': '#7387CE',
     FastAPI: '#F6BF00',
+    NextUI: '#A8DADC',
     'Next.js': '#B17CA6',
+    'Pragmatic Drag & Drop': '#FF6F61',
     Python: '#64B550',
     React: '#ED8C9B',
+    Requests: '#FFD166',
```
Comment (phoenixpereira) [inline src/constants/colours.ts:34]: This colour is to similar to the colour used for FastAPI, can you please change the colour. The dark blue colour you used previously would be fine.

File: src/constants/colours.ts [hunk 4]
```diff
@@ -22,15 +22,20 @@ export const BORDER_COLOURS = {
...
     Supabase: '#5DBBB5',
     'Tailwind CSS': '#907FC3',
     TypeScript: '#ACCB00',
+    Vite: '#646464',
```
Comment (phoenixpereira) [inline src/constants/colours.ts:38]: Can you change this to a colour beside grey, perhaps light blue or orange.

**Review by phoenixpereira — CHANGES_REQUESTED** [review]

**Review by phoenixpereira — APPROVED** [review]

File: .gitignore [hunk 5]
```diff
@@ -38,4 +38,4 @@ next-env.d.ts
 
 # SQLite dev database
 *.db
-*.sqlite
+*.sqlite
```
Comment (rayokamoto) [inline .gitignore:41]:
```suggestion
*.sqlite

```

**Review by rayokamoto — CHANGES_REQUESTED** [review]

**Review by rayokamoto — APPROVED** [review]

**Pull request merged**